src/components/EditSections/EditContactInfo/EditContactInfo.js

```diff
@@ -50,6 +51,8 @@ const EditContactInfo = ({
     },
   };
 
+  const displaypreferredEmailCommunications = Boolean(stripes.hasInterface('users', '16.2'));
```

Comment:
```suggestion
  const displayPreferredEmailCommunications = Boolean(stripes.hasInterface('users', '16.2'));
```
Use camel case here please